### PR TITLE
Add orders, products, and SKUs to the API.

### PR DESCRIPTION
**Pull request by rasmusrygaard**
r? @siddarth @spolu 
cc @brandur

### CODE REVIEW

**Comment by brandur** [comment]
You guys rock! Let me know if you need an additional review or anything ...

Fixes #149. /cc @steadicat

**Comment by rasmusrygaard** [comment]
@brandur If you have the bandwidth to look over this quickly that would be great. My main concern is that this has lived long enough internally that we've come to accept any quirks that may exist.

**Comment by brandur** [comment]
@rasmusrygaard Cool! Will take a look in a bit.

> My main concern is that this has lived long enough internally that we've come to accept any quirks that may exist.

Yeah :/ Unfortunately, as with some of the other modules that have been added here, we may not be able to catch all the quirks that come up on the initial pass, but we'll try to handle them appropriately as they're discovered.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.test
```
Comment (spolu) [inline .gitignore]: Surprised to see this part of that PR?

**Comment by spolu** [comment]
(otherwise LGTM as this has been pretty well tested in our own deployments)

File: .gitignore (hunk 1 above)
Comment (rasmusrygaard) [inline .gitignore]: Happy to open something separate for this, but it doesn't seem like there's any .gitignore in place at the moment

Comment (spolu) [inline .gitignore]: I'm completely fine with it :)

**Comment by brandur** [comment]
Looks good as far as I can tell! Will wait for a pass by @siddarth, but +1 for merging. Thanks again.

**Comment by rasmusrygaard** [comment]
Going to :ship: here as @siddarth already did a pass while this was used internally

**Pull request merged**